csbpg/public_schema.go

```diff
@@ -0,0 +1,22 @@
+package csbpg
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/lib/pq"
+)
+
+func relaxPublicSchemaRestrictions(tx *sql.Tx, cf connectionFactory) error {
+	log.Println("[DEBUG] make admin user owner of the public schema")
```

Comment:
informational: log.Print* are not the preferred way of logging inside providers. https://developer.hashicorp.com/terraform/plugin/log/managing

Reply:
Didn't know that. Thanks! I suggest we leave it like this.
log.Println is scattered all over the place in places I have not touched in this PR.
https://github.com/cloudfoundry/terraform-provider-csbpg/blob/351d269dde8a45b66baacb5f018266550bc81042/csbpg/resource_binding_user.go#L127

We can do a massive replace of all existing occurrences in a different PR, though.